mmv1/products/iap/WebCloudRunService.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'WebCloudRunService'
+description: |
+  Only used to generate IAM resources
+# This resource is only used to generate IAM resources. They do not correspond to real
+# GCP resources, and should not be used to generate anything other than IAM support.
+exclude_resource: true
+docs:
+id_format: 'projects/{{project}}/iap_web/cloud_run-{{location}}/services/{{name}}'
+base_url: 'projects/{{project}}/iap_web/cloud_run-{{location}}/services/{{name}}'
+self_link: 'projects/{{project}}/iap_web/cloud_run-{{location}}/services/{{name}}'
+import_format:
+  - 'projects/{{project}}/iap_web/cloud_run-{{location}}/services/{{name}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+iam_policy:
+  method_name_separator: ':'
+  parent_resource_type: 'google_cloud_run_v2_service'
+  fetch_iam_policy_verb: 'POST'
+  allowed_iam_role: 'roles/iap.httpsResourceAccessor'
+  parent_resource_attribute: 'name'
+  iam_conditions_request_type: 'REQUEST_BODY'
+  example_config_body: 'templates/terraform/iam/iam_attributes.go.tmpl'
+  import_format:
+    - 'projects/{{project}}/iap_web/cloud_run-{{location}}/services/{{name}}'
+    - '{{name}}'
+  min_version: beta
```

Comment:
qq: Could you clarify the reasoning for marking this resource as `beta-only`? I noticed the beta `base_url` points to the `v1` endpoint, which is the same as GA.

Reply:
I marked the resource beta because IAP for Cloud Run is preview feature, but I don't want to break existing beta provider users' compatibility.

Reply:
Sorry for missing this comment. Regarding the preview feature, could you check go/terraform-launch-stages to confirm the correct provider (beta vs. private)? Thanks!